internal/testrunner/runners/system/servicedeployer/terraform_test.go

```diff
@@ -121,7 +143,8 @@ func TestAddTerraformOutputs(t *testing.T) {
 			}
 
 			// Test that the terraform output values are generated correctly
-			addTerraformOutputs(tc.ctxt)
+			err := addTerraformOutputs(tc.ctxt)
+			assert.Equal(t, err, nil)
```

Comment:
nit. there is a specific method to test the errors:

```suggestion
			assert.NoError(t, err)
```

or maybe it can be required to not be an error there 

```suggestion
			require.NoError(t, err)
```